src/index.html

```diff
@@ -16,17 +19,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      class="page__form"
+      action="#"
+      data-qa="big"
+    >
```

Comment:
The attributes of the `<form>` tag should be placed on new lines, each indented by 2 spaces relative to the tag, to keep the attributes correctly formatted.